starter-code/public/scripts/controllers/articleController.js

```diff
@@ -3,10 +3,14 @@ var app = app || {};
 
 (function(module) {
   const articleController = {};
-  // TODO: Setup a function that kicks off the fetching and rendering of articles, using the same
+  // Done!TODO: Setup a function that kicks off the fetching and rendering of articles, using the same
   // code that used to be in index.html.
   // Also be sure to hide all the main section elements, and reveal the #articles section:
-
+  articleController.init = function () {
+    $('.tab-content').hide();
+    $('#articles').show();
+  }
+  app.Article.fetchAll();
```

Comment:
Since this should be part of the init function call, it should be inside the function. Also, if you look at fetchAll, it requires a callback. So we need a callback inside of our fetchAll call.